Packs/FeedFastly/Integrations/FeedFastly/FeedFastly.py

```diff
@@ -0,0 +1,31 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+from JSONFeedApiModule import *  # noqa: E402
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+    params['indicator'] = 'ip'
+    params['url'] = 'https://api.fastly.com/public-ip-list'
+
+    indicator_types = [
+        {
+            'indicator_type': FeedIndicatorType.CIDR,
+            'extractor': 'addresses[].{ip:@}'
+        },
+        {
+            'indicator_type': FeedIndicatorType.IPv6CIDR,
+            'extractor': 'ipv6_addresses[].{ip:@}'
+        }
+    ]
+
+    for item in indicator_types:
+        params['indicator_type'] = item['indicator_type']
+        params['extractor'] = item['extractor']
+        feed_main(params, 'Fastly Feed', 'fastly')
```

Comment:
We're calling `feed_main` twice? Once for each indicator type - this seems silly. There's got to be a better way to do this.